README.md

```diff
@@ -412,16 +412,18 @@ Are you using this project or any of our other projects? Consider [leaving a tes
-- [terraform-aws-ec2-autoscale-group](https://github.com/cloudposse/terraform-aws-ec2-autoscale-group) - Terraform module to provision Auto Scaling Group and Launch Template on AWS
-- [terraform-aws-ecs-container-definition](https://github.com/cloudposse/terraform-aws-ecs-container-definition) - Terraform module to generate well-formed JSON documents (container definitions) that are passed to the  aws_ecs_task_definition Terraform resource
-- [terraform-aws-ecs-alb-service-task](https://github.com/cloudposse/terraform-aws-ecs-alb-service-task) - Terraform module which implements an ECS service which exposes a web service via ALB
-- [terraform-aws-ecs-web-app](https://github.com/cloudposse/terraform-aws-ecs-web-app) - Terraform module that implements a web app on ECS and supports autoscaling, CI/CD, monitoring, ALB integration, and much more
-- [terraform-aws-ecs-codepipeline](https://github.com/cloudposse/terraform-aws-ecs-codepipeline) - Terraform module for CI/CD with AWS Code Pipeline and Code Build for ECS
-- [terraform-aws-ecs-cloudwatch-autoscaling](https://github.com/cloudposse/terraform-aws-ecs-cloudwatch-autoscaling) - Terraform module to autoscale ECS Service based on CloudWatch metrics
-- [terraform-aws-ecs-cloudwatch-sns-alarms](https://github.com/cloudposse/terraform-aws-ecs-cloudwatch-sns-alarms) - Terraform module to create CloudWatch Alarms on ECS Service level metrics
-- [terraform-aws-ec2-instance](https://github.com/cloudposse/terraform-aws-ec2-instance) - Terraform module for providing a general purpose EC2 instance
-- [terraform-aws-ec2-instance-group](https://github.com/cloudposse/terraform-aws-ec2-instance-group) - Terraform module for provisioning multiple general purpose EC2 hosts for stateful applications
+- [terraform-aws-eks-fargate-profile](https://github.com/cloudposse/terraform-aws-eks-fargate-profile) - Terraform module to provision an EKS Fargate Profile
+- [terraform-aws-eks-node-group](https://github.com/cloudposse/terraform-aws-eks-node-group) - Terraform module to provision an EKS Node Group
+- [terraform-aws-eks-iam-role](https://github.com/cloudposse/terraform-aws-eks-iam-role) - Terraform module to provision an EKS IAM Role for Service Account
+- [terraform-aws-eks-spotinst-ocean-nodepool](https://github.com/cloudposse/terraform-aws-eks-spotinst-ocean-nodepool) - A terraform module to create an EKS node pool with Spot Ocean by Spot.io
+- [terraform-provider-aws](https://github.com/hashicorp/terraform-provider-aws) - terraform-provider-aws
+- [terraform-provider-kubernetes](https://github.com/hashicorp/terraform-provider-kubernetes) - terraform-provider-kubernetes
+- [terraform-provider-local](https://github.com/hashicorp/terraform-provider-local) - terraform-provider-local
+- [terraform-provider-null](https://github.com/hashicorp/terraform-provider-null) - terraform-provider-null
+- [terraform-provider-template](https://github.com/hashicorp/terraform-provider-template) - terraform-provider-template
```

Comment:
@SweetOps Providers should go under `references`; ideally, we'd link to the terraform registry documentation for the provider instead.